pubspec.yaml

```diff
@@ -61,6 +61,7 @@ dependencies:
   rubber: ^0.4.0
   path_provider: ^1.6.24
   dotted_border: ^1.0.6
+  pedantic: ^1.9.2
```

Comment:
this is widely used in the Industry. is good, is helpful, and we should use it too.